compose/compose-tooling/src/main/java/com/squareup/workflow/ui/compose/tooling/ViewFactories.kt

```diff
@@ -51,3 +44,27 @@ import com.squareup.workflow.ui.compose.WorkflowRendering
     previewViewEnvironment(placeholderModifier, viewEnvironmentUpdater, mainFactory = this)
   WorkflowRendering(rendering, previewEnvironment, modifier)
 }
+
+@OptIn(WorkflowUiExperimentalApi::class)
+@Preview(showBackground = true)
+@Composable private fun ViewFactoryPreviewPreview() {
```

Comment:
`ViewFactoryPreviewPreview` -> `ComposedViewFactoryPreview`?

Reply:
This isn't just for composed view factories, it works for any view factory.